setup.py

```diff
@@ -0,0 +1,23 @@
+import setuptools
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name="qtmcop-docs",
+    version="0.0.1",
+    author="Department of Transport and Main Roads",
+    author_email="nikhil.z.ahuja@tmr.qld.gov.au",
```

Comment:
Use this generic one instead of your own? It does get monitored by Jamie and other LT I believe.
```suggestion
    author_email="transport_analysis_requests@tmr.qld.gov.au",
```

Reply:
Oh this is in the setup - it wouldn't show up in the docs - but yeah makes sense to not put mine anyway